ibis/expr/api.py

```diff
@@ -124,6 +125,7 @@
     'join',
     'least',
     'literal',
+    'lit',
```

Comment:
Can you add a quick test for it so we don't accidentally remove it without notice?